pkgs/games/wolfstoneextract/default.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, stdenv
+, fetchFromBitbucket
+, cmake
+}:
+stdenv.mkDerivation rec {
```

```suggestion
}:

stdenv.mkDerivation rec {
```
please add a blank line here.